src/LobbyServer.hs

```diff
@@ -29,11 +31,18 @@ import Data.List
 import Data.Maybe
 import LobbyTypes
 import Hastings.Utils
+
```

Comment:
Organize imports.
Divide into imports from our project and import from other libraries